cli/prove.ts

```diff
@@ -53,15 +63,46 @@ export const prove = async (
             dataQuery,
         }
 
+        let axiomV2QueryAddress;
+        if (blockhashOracle) {
+            if (broadcaster) {
+                throw new Error("Cannot use both broadcaster and blockhash oracle");
+            }
+            if (!targetChainId) {
+                throw new Error("`targetChainId` is required for blockhash oracle bridge type");
+            }
+            axiomV2QueryAddress = getAxiomV2QueryBlockhashOracleAddress(sourceChainId, targetChainId);
+        } else if (broadcaster) {
+            if (!targetChainId) {
+                throw new Error("`targetChainId` is required for broadcaster bridge type");
+            }
+            if (!bridgeId) {
+                throw new Error("`bridgeId` is required for broadcaster bridge type");
+            }
+            axiomV2QueryAddress = getAxiomV2QueryBroadcasterAddress(sourceChainId, targetChainId, bridgeId);
+        } else {
+            axiomV2QueryAddress = getAxiomV2QueryAddress(sourceChainId);
+        }
+
+        let target;
+        if (blockhashOracle || broadcaster) {
+            let targetRpcUrlOrCache = getRpcUrl(targetRpcUrl!);
```

Comment:
nit: this can be `const`

Reply:
other nit: targetRpcUrl can be passed in without `!` since `getRpcUrl` takes in an argument of type `string | undefined` so the types should match

```
function getRpcUrl(rpcUrl: string | undefined): string
```